### ISSUE TITLE
Sort fields alphabetically (or through some other order)

### ISSUE THREAD

**Issue by kinow** [common-workflow-language/cwl-v1.2#187]
There are some lists where `id` appears first, but others where it appears in the middle of the list of fields, e.g. 

`Subworkflows`

![image](https://user-images.githubusercontent.com/304786/172135423-24ab545b-10b8-458c-b42c-c854cff2f7ab.png)

`CommandInputParameter`

![image](https://user-images.githubusercontent.com/304786/172135228-82c0221c-0b27-48e5-8e9e-380b085aac2c.png)

And the fields are not following any order. IMO it would be helpful to readers to sort the fields somehow. I think it could be either simply alphabetically, or leave some important items at the top (e.g. id, doc) and the rest alphabetically.

**Comment by mr-c** [comment]
Yes. Alphabetical can be default, but I want a method to force a particular order (inclusive of inherited fields)

We also need the ability to link to a specific field
